contrib/tig-completion.bash

```diff
@@ -93,13 +96,15 @@ if [ -n "$ZSH_VERSION" ]; then
 	bashcompinit
 fi
 
-# we use internal git-completion functions, so wrap _tig for all necessary
+# we use internal git-completion functions (if available), so wrap _tig for all necessary
 # variables (like cword and prev) to be defined
-__git_complete tig _tig 
+if [ "$(type -t __git_complete)" = function ]; then
```

Comment:
Nitpick... `if ! declare -f __git_complete &>/dev/null; then` is a better test for existence of function because it doesn't spawn shell with `$()`